src/mcts/search.cc

```diff
@@ -880,7 +877,7 @@ void SearchWorker::DoBackupUpdate() {
 // 7. UpdateCounters()
 //~~~~~~~~~~~~~~~~~~~~
 void SearchWorker::UpdateCounters() {
-  search_->UpdateRemainingMoves();  // Updates smart pruning counters.
+  search_->UpdateRemainingMoves(); // Updates smart pruning counters.
```

Comment:
This change is unrelated.  I believe that this codebase is following google c++ style - and two spaces is the correct whitespace in this scenario under that style guide.

Reply:
er... why does it say that lol